terragrunt/modules/ecs/templates/task-definitions/organisation-app.json.tftpl

```diff
@@ -37,6 +37,8 @@
         {"name": "OrganisationService", "value": "https://organisation.${public_domain}"},
         {"name": "Organisation__Authority", "value": "https://authority.${public_domain}"},
         {"name": "PersonService", "value": "https://person.${public_domain}"},
+        {"name": "SharedSessions", "value": "${shared_sessions_enabled}"},
```

Comment:
Shared sessions is a feature flag, so should be something like `Features__SharedSessions`. @mareg please confirm.

Related app change: https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/pull/971/files#diff-efd1dcae149754760876b9e248c8b832b93447145deca248bf3e35ea08efe348